FlowCrypt/Core/Core.swift

```diff
@@ -7,7 +7,6 @@ import JavaScriptCore
 enum CoreError: Error {
     case exception(String)
     case notReady(String)
-    case undefined
```

Comment:
Not sure what `undefined` error would be for, the `Core` library does not need it.